tests/cpu/PathUtils_tests.cpp

```diff
@@ -15,7 +16,22 @@ OCIO_ADD_TEST(PathUtils, compute_hash)
     const std::string file1 = OCIO::GetTestFilesDir() + "/lut1d_4.spi1d";
     const std::string file2 = OCIO::GetTestFilesDir() + "/lut1d_5.spi1d";
 
-    OCIO_CHECK_EQUAL(OCIO::ComputeHash(file1), OCIO::ComputeHash(file1));
+    OCIO_CHECK_EQUAL(OCIO::g_hashFunction(file1), OCIO::g_hashFunction(file1));
 
-    OCIO_CHECK_NE(OCIO::ComputeHash(file1), OCIO::ComputeHash(file2));
+    OCIO_CHECK_NE(OCIO::g_hashFunction(file1), OCIO::g_hashFunction(file2));
+
+    const std::string result1 = OCIO::g_hashFunction(file1);
+    const std::string result2 = OCIO::g_hashFunction(file2);
+
+    {
+        OCIO::g_hashFunction = OCIO::CustomComputeHash;
```

Comment:
You should use the public method to change the method implementation .

Reply:
Working on this sir.